keel-orca/keel-orca.gradle

```diff
@@ -19,11 +19,11 @@ apply plugin: "java-library"
 dependencies {
   implementation project(":keel-retrofit")
   implementation project(":keel-clouddriver")
-  implementation "com.fasterxml.jackson.core:jackson-databind:[2.9.6,)"
-  implementation "com.fasterxml.jackson.core:jackson-annotations:[2.9.6,)"
+  implementation "com.fasterxml.jackson.core:jackson-databind"
+  implementation "com.fasterxml.jackson.core:jackson-annotations"
   implementation "org.jetbrains.kotlin:kotlin-reflect"
 
-  api "com.fasterxml.jackson.module:jackson-module-kotlin:[2.9.6,)"
-  api "com.github.jonpeterson:jackson-module-model-versioning:+"
-  api "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:[2.9.6,)"
+  api "com.fasterxml.jackson.module:jackson-module-kotlin"
+  api "com.github.jonpeterson:jackson-module-model-versioning:latest.release"
```

Comment:
I'd be hesitant about putting `latest.release` on anything without some sort of locking in place because it makes the builds unstable and also noticably slower, but especially something considered part of the module api

Reply:
Yeah, I'll do that. I was being lazy.

Reply:
Specifically with `retrofit2-kotlin-coroutines-adapter` that won't be necessary for long as there's an approved PR in Retrofit to get that capability merged in core.